triggers/service/src/main/scala/com/digitalasset/daml/lf/engine/trigger/Cli.scala

```diff
@@ -323,10 +323,19 @@ private[trigger] object Cli {
       .action {
         case ("json", cli) => cli.copy(logEncoder = LogEncoder.Json)
         case ("plain", cli) => cli.copy(logEncoder = LogEncoder.Plain)
-        case (other, _) =>
-          throw new IllegalArgumentException(s"Unsupported logging encoder $other")
+        case (other, _) => throw new IllegalArgumentException(s"Unsupported logging encoder $other")
       }
 
+    opt[Long]("max-batch-size")
+      .optional()
+      .text(
+        s"maximal number of message processed between two high-level rule triggers. Defaults to ${DefaultTriggerRunnerConfig.maximumBatchSize}"
```

Comment:
Nit pick: maybe write "maximum number of messages..." instead?

Reply:
:+1: